tests/invoke_test.go

```diff
@@ -0,0 +1,43 @@
+package tests
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestInvokeVM(t *testing.T) {
+
+	var stdout bytes.Buffer
+	var stderr bytes.Buffer
+	setupCmd := exec.Command("invoke", "setup")
+	setupCmd.WaitDelay = 5 * time.Second
```

Comment:
Can you explain the purpose of this line?

Reply:
Some tests I did, but it is useless now. Removing it